src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -114,13 +114,13 @@ def get_resolution_from_duration(duration: timedelta) -> int:
 
 def get_target_name(series: PrometheusSeries) -> Optional[str]:
     for label in ["container", "pod", "node"]:
-        if label in series.metric:
-            return series.metric[label]
+        if label in series["metric"]:
```

Comment:
Why do we need this?
`PrometheusSeries` isn't a dict
It does have a field named `metric`

Reply:
typing is unclear it is a dict, fixing